packages/vx-responsive/src/enhancers/withParentSize.js

```diff
@@ -28,7 +28,7 @@ export default function withParentSize(BaseComponent) {
 
     resize(event) {
       if (this.container) {
-        var boundingRect = this.container.getBoundingClientRect();
+        const boundingRect = this.container.getBoundingClientRect();
```

Comment:
I think you can still use const here since the closure is contained within the same closure that `var` is defined in.

In general, I think `let` and `const` work the same way most variable assignments work in other languages, but `var` is the nasty one :'( 

[There's a good SE question about it here.](https://softwareengineering.stackexchange.com/questions/274342/is-there-any-reason-to-use-the-var-keyword-in-es6)